### PR TITLE
Guidelines for writing CSS in AMP HTML

### PR DESCRIPTION
**Pull request by camelburrito**
guidelines for creating new CSS  or editing existing in AMP HTML

### CODE REVIEW

**Comment by camelburrito** [comment]
I signed it!

File: contributing/writing-css.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+In general, the order of the CSS selectors do not affect which rules get applied to your CSS,
+unless the selectors have the same specificity and apply to at-least one element in common.
+A well written CSS will work perfectly well even when the selectors are re-ordered (example
+scenario- reordering imports in a css file). In real world it is hard to achieve this , especially
+when we import CSS via gulp or 3p css (which can change at any time)
+
+### How to calculate Specificity
+
+This blog explains specificty in very simple terms and helps to understand how it is computed -
+https://css-tricks.com/specifics-on-css-specificity/​ (very short and understandable - I promise)
```
Comment (SchnWalter) [inline contributing/writing-css.md]: Clicking the "View" button, to see the compiled version of this markdown file, you will see that this link points to `https://css-tricks.com/specifics-on-css-specificity/%E2%80%8B` (and the site redirects you to another page)

File: contributing/writing-css.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  - [FOUC](#fouc)
+  - [Rule of thumb](#rule-of-thumb)
+- [Case Studies](#case-studies)
+  - [AMP Selector CSS has high CSS specificity](#amp-selector-css-has-high-css-specificity)
+  - [AMP Accordion CSS Changes broke specificity](#amp-accordion-css-changes-broke-specificity)
+- [Reading material](#reading-material)
+
+## Who should read this doc?
+
+Are you a contributor to ampproject/amphtml, so you make changes to the runtime or add or
```
Comment (bpaduch) [inline contributing/writing-css.md]: Either:

- Are you a contributor to ampproject/amphtml? Do you make changes to the AMP Runtime or add/modify AMP extensions? If you answer yes, then you should definitely read this document.
- Do you contribute changes to the AMP Runtime or add/modify AMP extensions? If you answer yes, then you should definitely read this document.

**Comment by camelburrito** [comment]
Closing since CLA does not pass - please follow #10743

**Pull request closed without merging**